README.md

```diff
@@ -268,6 +268,21 @@ $ remme node-account transfer-tokens-from-frozen-to-unfrozen
 }
 ```
 
+Transfer available tokens from unfrozen reputational balance to operational balance (executable only on the machine which runs the node) — `remme node-account transfer-tokens-from-unfrozen-to-operational --amount=1000`.
```

Comment:
We never write argument here. So `remme node-account transfer-tokens-from-unfrozen-to-operational --amount=1000` to `remme node-account transfer-tokens-from-unfrozen-to-operational`.

Reply:
Done.